src/Apps/Artist/Routes/Overview/index.tsx

```diff
@@ -255,6 +266,9 @@ export const OverviewRouteFragmentContainer = createFragmentContainer(
           }
         }
       }
+      insights {
```

Comment:
Ah got it, so this is just used for an existy check (if the artist _just_ has these insights).